### ISSUE TITLE
Add a citation.cff file

### ISSUE THREAD

**Issue by ElenaFdR** [publiccodenet/standard#303]
[Academic software developers recommend](https://twitter.com/matkuzak/status/1223273664407314432) [a .cff file](https://citation-file-format.github.io/) as best practice for all software projects.

**Comment by ElenaFdR** [comment]
And as a meta question, should this be added to the Standard criteria?

**Comment by ericherman** [comment]
I don't think so. With a few exceptions, the standard should not typically specify a technology (e.g.: BibTeX or CFF). Perhaps a MAY (or maybe SHOULD) guideline around "easy to correctly cite" might be worth thinking about, if a strong argument can be made that it would increase the reuse of a codebase.

**Comment by Ainali** [comment]
I was just helping out adding comments to a paper-in-writing about making code last and found myself wanting to reference the Standard. Is that enough to reopen this?

**Comment by ericherman** [comment]
As you have a concrete example, perhaps create a new issue stating your use case.
(Naturally, mention this closed issue in that use case.)

Do you want to reference the standard as a book or as an online resource?

Given the still evolving state, I think that citing it as a book might be odd, since it isn't published as a book with an ISBN and such.

Looking at some book citiation examples like:

``` bibtex
@book{einstein1921,
    title        = {Relativity: The Special and the General Theory},
    author       = {Einstein, A.},
    year         = 1920,
    publisher    = {Henry Holt and Company},
    address      = {London, United Kingdom},
    isbn         = 9781587340925
}
```
Most of those fields do not have values (but might be optional), and the `author` field would be very, very long.

As a book, I feel like it's still in a pre-published draft state.

By contrast, looking at an example of citing as an online resource, this looks like it might be better:

``` bibtex
@misc{WinNT,
  title = {{MS Windows NT} Kernel Description},
  howpublished = {\url{http://web.archive.org/web/20080207010024/http://www.808multimedia.com/winnt/kernel.htm}},
  note = {Accessed: 2010-09-30}
}
```

But, naturally, it really depends on what your use case is.
